C/CUDA/CUTENSOR/build_tarballs.jl

```diff
@@ -8,62 +8,40 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
 
 name = "CUTENSOR"
-version = v"2.2.0"
+version = v"2.3.0"
 
+script_builder(version::String) = """
```

Comment:
Just looking at this script, does it use the `version` argument anymore? If it doesn't, I'd suggest removing it and switching back to a raw string (and remove the escaping on the dollar signs).